plugins/newrelic-dashboard/src/types/DashboardEntity.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type DashboardEntity = {
+  data: {
+    actor: {
+      entitySearch: {
+        results: {
+          entities: [ResultEntity];
```

Comment:
wait, this means an array of one ResultEntity, you meant `Array<ResultEntity>` or `ResultEntity[]` right?